CHANGELOG.rst

```diff
@@ -623,6 +623,11 @@ General
   documentation have been fixed and now point to the correct source files on
   GitHub.
 
+- Reorder :file:`bootstrap.yml` Ansible playbook to also work for systems freshly
+  installed from CD. :ref:`debops.apt` needs to be run early to regenerate
+  :file:`/etc/apt/sources.list` which might still contain a now not functional
+  CD entry.
+
```

Comment:
Looks like this entry is way, way down the Changelog file, in a completely wrong release. It's late so I'll merge the PR tommorow; if you want you could move this entry to the correct place, I'd say in the Changed section. For that you probably woul have to rebase the commits on latest `master` as well. If not, I can do that tommorow.